README.md

```diff
@@ -64,27 +64,27 @@ Additionally, while CLIP1 and 2 both gave the top one classification prediction
 provide the two highest value predictions. This 'second most likely class' in the CLIP3 data and the above information on categories and keywords make CLIP3 ideal for further analysis. The results could shed light on specifications desirable for work with similar Enlightenment era book illustrations.
 
 In the Hackathon outcomes, only classification entries with min. 50% accuracy for the first candidate were included into the
-analyses and subsequent interpretation. However, the accurracy span was wide in all three runs. In this task, I summarise these tendencies, verify them with close-reading, and speculate on their possible reasons and outcomes. I do this on the data about the third run (CLIP3), for several reasons:
+analyses and subsequent interpretation. However, the accurracy span was wide in all three runs. In this task, I summarise these tendencies, verify them with close-reading, and speculate on their possible reasons and outcomes. I do this on the data about the third run (CLIP3), for the following reasons:
 - the data from this run also provides the second most probable category the model assigned to each entry, which can shed light on the model's behaviour with this type of a dataset
 - the model used in the third run was already customised based on human-annotated feedback on the previous two
 
 ### Primary data
 A spreadsheet containing (per image):
 - its URL (from a database containing all images from the dataset; for qualitative close-reading analysis)
 - CLIP3 first-choice category and the certainty score of that classification (score is based on training data; for CLIP3, this is human-produced image classification annotations)
-- CLIP3 second-choice category and the certainty score of that classification (-,,-)
+- CLIP3 second-choice category and the certainty score of that classification
 
 ### Secondary data
-(used for refining the RQ and hypotheses; would recommend for exploration of H2, H3)
 A spreadsheet containing (per every image from CLIP3 first-choice):
 - its URL (see Primary data)
 - CLIP3 first-choice category (see Primary data)
-- evaluation score of that classification (i.e., a percentage score expressing the probability of a human annotator classifying an image as belonging in the same first-choice category as CLIP3 did; based on human-annotated control batch produced by the group)<br>
+- evaluation score of that classification (i.e., a percentage score expressing the probability of a human annotator classifying an image as belonging in the same first-choice category as CLIP3 did; based on human-annotated control batch produced by the group)
+
 Various visualisations based on the above spreadsheet (created during the hackathon)
```

Comment:
Hackathon is capitalized everywhere else, although personally I like it more lowercase